### ISSUE TITLE
Offset for the first labels

### ISSUE THREAD

**Issue by Elipasousa** [Dreem-Devices/DMPickerView#1]
Hi, I have a DMPickerView from 0 to 99. The offset of the first labels seems a little bit strange. From 3 to 99, everything works fine, but labels 0, 1 and 2 are cutted off. Do you know why this is happening? What am I doing wrong?

![1](https://cloud.githubusercontent.com/assets/11166768/16616680/9a1ec814-4376-11e6-9e9e-9820ad7de53b.png)

![2](https://cloud.githubusercontent.com/assets/11166768/16616681/9ea3af12-4376-11e6-878f-41bf2907653e.png)

(this is how I wanted all labels to appear)
![3](https://cloud.githubusercontent.com/assets/11166768/16616684/a37e3ffc-4376-11e6-9e74-bae5f9613738.png)

**Comment by Elipasousa** [comment]
Hi again, I think I found the problem: on your `layoutSubviews` method, you're calculating the frame for all labels, according to their initial position. As labels 1, 2, 3, ..., etc, are scaled down, width and height are smaller. You're calculating the Y position (in VERTICAL case) assuming this smaller frame. The frame of the label should always be the frame as if the label is selected, regardless it's scale down or not. I managed to get this working by overriding the frame's height on `layoutSubviews` (for the VERTICAL picker)
